### PR TITLE
Sdd 127 add azure maps app settings v3

### PR DESCRIPTION
**Pull request by thelious1974**

### CODE REVIEW

File: Resources/test-parameters.json [hunk 1]
```diff
@@ -39,7 +39,10 @@
             "value": "not-a-real-key"
         },
         "templateBaseUrl": {
-            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-employmentprogression/master-v3/Resources/"
+            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-employmentprogression/SDD-127-AddAzureMapsAppSettings-v3/Resources/"
```
Comment (thelious1974) [inline Resources/test-parameters.json]: I'll need to set this back to "master" after this is merged,  unfortunately.. it seems that updating the local nested template in step with this doesn't have a great story.

File: Resources/template.json [hunk 2]
```diff
@@ -40,6 +40,9 @@
         },
         "templateBaseUrl": {
             "type": "string"
+        },
+        "azureMapSubscriptionKey": {
+            "type": "string"
```
Comment (rivoric) [inline Resources/template.json]: Should this be a string or securestring?

File: Resources/deployments/function-app-resources.json [hunk 3]
```diff
@@ -34,6 +34,9 @@
         },
         "storageAccountKey": {
             "type": "string"
+        },
+        "azureMapSubscriptionKey": {
+            "type": "string"
```
Comment (rivoric) [inline Resources/deployments/function-app-resources.json]: Should this be a string or securestring?

File: Resources/template.json (hunk 2 above)
Comment (thelious1974) [reply Resources/template.json]: changed to securestring

**Pull request merged**